examples/example.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const { createReadStream } = require('fs')
+const { resolve } = require('path')
+const { fastifyThrottle } = require('../index')
+const { RandomStream } = require('../test/utils/random-stream')
+
+async function main () {
+  const fastify = require('fastify')()
+
+  await fastify.register(fastifyThrottle, {
+    bps: 10000,
+    streamPayloads: true,
+    stringPayloads: true,
+    bufferPayloads: true
+  })
+
+  fastify.get('/string', (req, reply) => {
+    reply.send(Buffer.allocUnsafe(1024 * 1024).toString('ascii'))
+  })
+
+  fastify.get('/buffer', (req, reply) => {
+    reply.send(Buffer.allocUnsafe(1024 * 1024))
+  })
+
+  fastify.get('/stream', (req, reply) => {
+    reply.send(new RandomStream(30000))
+  })
+
+  fastify.get('/delayed', {
+    config: {
+      throttle: {
+        bps: function (elapsedTime, bytes) {
+          if (elapsedTime < 2) {
+            return 0
+          } else {
```

Comment:
else is redundant, you already return in the first branch

Reply:
I know, but I like the formatting and it is ok for an example. I would like to consider this a nitpick